### ISSUE TITLE
Add libuv to the Microsoft.NETCore.App redist

### ISSUE THREAD

**Issue by joshfree** [dotnet/cli#2277]
Moved from https://github.com/dotnet/corefx/issues/7481 on behalf of @davidfowl 
> Reasoning behind this is because it's a purely native component with 0 managed surface area. We now have 6 native libuv assets in the kestrel package:

> ![image](https://cloud.githubusercontent.com/assets/95136/14239035/7f65cfc8-f9ec-11e5-91c7-24d89626b552.png)

> Today, every ASP.NET application depends on this and removing it would reduce the output size by 2MB. We'd make a libuv package and make kestrel depend on it so that it would be possible to use a newer version that what was shipped in the redist (like we do for every other component).

David let us know when you have the libuv package created so we can chain it into the redist

**Comment by piotrpMSFT** [comment]
@weshaggard what are your thoughts?

**Comment by weshaggard** [comment]
I think we are landing on adding this but I suspect it will be post-RC2 so the RTM milestone looks correct. We of course do need the packages before we can actually do this. @davidfowl is there another issue somewhere tracking the creation of the libuv packages?

**Comment by muratg** [comment]
https://github.com/aspnet/KestrelHttpServer/issues/730

**Comment by halter73** [comment]
We have created a Libuv 1.9.0 package which is currently being published to myget.org/F/aspnetrelease/ and will be published to nuget.org for RC2. We would still like to include this in the redist for RTM.

**Comment by muratg** [comment]
:eyes:
